validate_states.py

```diff
@@ -195,6 +196,15 @@ def get_telem_values(tstart, msids, days=14, dt=32.8, name_map={}):
     return out
 
 
+def get_bad_mask(tlm):
+    mask = np.zeros(len(tlm), dtype='bool')
+    for interval in characteristics.bad_times:
+        bad = ((tlm['date'] >= DateTime(interval['start']).secs)
```

Comment:
Not a big deal here, but in general if you are getting intervals from sorted time lists you should use np.searchsorted like
```
i0, i1 = np.searchsorted(tlm['date'], [tstart, tstop])
mask[i0:i1] = True
```
This is way faster, although I confess that I always have to think about off-by-one issues at the edges.  Even here I'm not 100% sure if what I wrote is exactly right so you can leave this if you want.  But when performance is an issue...